packages/babel-plugin-transform-modules-commonjs/src/index.ts

```diff
@@ -127,7 +129,7 @@ export default declare((api, options: Options) => {
     AssignmentExpression(path) {
       const left = path.get("left");
       if (left.isIdentifier()) {
-        const localName = path.node.name;
+        const localName = left.node.name;
```

Comment:
This is a bug fix. Babel does not replace `module` in `module = {}` to assertion errors because `path.node.name` is always `undefined`.